influxdb/build.gradle

```diff
@@ -1,11 +1,20 @@
 version = '0.1-SNAPSHOT'
 
 dependencies {
+    def hazelcastVersion = "3.12"
+    def jetVersion = "3.0"
     compile 'org.influxdb:influxdb-java:2.15'
     testCompile "org.testcontainers:testcontainers:1.11.2"
     testCompile "org.testcontainers:influxdb:1.11.2"
+    testCompile "com.hazelcast.jet:hazelcast-jet-core:" + jetVersion + ":tests"
+    testCompile "com.hazelcast:hazelcast:" + hazelcastVersion
+    testCompile "com.hazelcast:hazelcast:" + hazelcastVersion + ":tests"
+    testCompile "com.hazelcast:hazelcast-client:" + hazelcastVersion + ":tests"
+    testCompile group: 'javax.cache', name: 'cache-api', version: '1.1.0'
```

Comment:
we could probably move all the test deps to the root, since I imagine the other modules will need it too?